extensions/wikia/Wall/js/Wall.js

```diff
@@ -302,15 +302,20 @@
 		vote: function (e) {
 			e.preventDefault();
 			if (!window.wgUserName) {
-				window.wikiaAuthModal.load({
-					forceLogin: true,
-					origin: 'wall-and-forum',
-					onAuthSuccess: function () {
-						this.voteBase(e, function () {
-							window.location.reload();
-						});
-					}.bind(this)
-				});
+				if (!this.isMonobook) {
+					window.wikiaAuthModal.load({
```

Comment:
I think these should be consistent and preferably use `require(['AuthModal'], function (authModal) {` like below.